sample-compose/src/main/java/com/airbnb/lottie/sample/compose/examples/AnimatableExamplesPage.kt

```diff
@@ -0,0 +1,110 @@
+package com.airbnb.lottie.sample.compose.examples
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Slider
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import com.airbnb.lottie.compose.*
+import com.airbnb.lottie.sample.compose.R
+
+@Composable
+fun AnimatableExamplesPage() {
+    UsageExamplePageScaffold {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .verticalScroll(rememberScrollState())
+        ) {
+            ExampleCard("Example 1", "Repeat Forever") {
+                Example1()
+            }
+            ExampleCard("Example 2", "Draggable Progress Slider") {
+                Example2()
+            }
+            ExampleCard("Example 3", "Draggable Speed Slider") {
+                Example3()
+            }
+        }
+    }
+}
+
+@Composable
+private fun Example1() {
+    val anim = remember { LottieAnimatable() }
+    val composition by lottieComposition(LottieCompositionSpec.RawRes(R.raw.heart))
+    LaunchedEffect(composition) {
+        anim.animate(
+            composition,
+            iterations = LottieConstants.IterateForever,
+        )
+    }
+    LottieAnimation(composition, anim.progress)
```

Comment:
This seems a little redundant and potentially easy to forget.  If a user has already created a lottie composition, and a LottieAnimatable, can the wire-up be done in either of those two steps instead of requiring an extra step? Animatable seems to have the info about both the composition, and progress.

Reply:
@doris4lt did reading the rest of the examples clarify this? The LottieAnimation overload handles this case.

Reply:
I get why this is needed. It is a 3-step setup that I recommend slimming down a bit for better ergonomic. 

Maybe consider combining the first two steps with:  `val (composition, anim) = lottieComposition(...)`

Reply:
@dors4lt I think 90% of people will be using the declarative `LottieAnimataion` composable. `LottieAnimatable` is also not coupled to a single composition so I don't think it makes sense to combine the two here. You can see one such example on the transitions examples page.